systems/spock_quick_start_guide.rst

```diff
@@ -282,6 +290,7 @@ To use GPU-aware Cray MPICH with the Cray compiler wrappers, users must load spe
     ## These must be set before running
     export MPIR_CVAR_GPU_EAGER_DEVICE_MEM=0
     export MPICH_GPU_SUPPORT_ENABLED=1
+    export MPICH_SMP_SINGLE_COPY_MODE=CMA
```

Comment:
For this variable, we should mention that it is needed temporarily due to a known issue as we do want people to remove this once the issue is fixed